README.md

```diff
@@ -9,117 +19,62 @@ This repository explores the application of XL-Net on user review based question
-source ~/.bash_profile
-```
-Create new development branch and switch onto it
-```
-branch_name=dev-readme_requisites-20180905 # Name of development branch, of the form 'dev-feature_name-date_of_creation'}}
-git checkout -b $branch_name
-```
 
-## Initial Commit
-Lets start with a blank slate: remove `.git` and re initialize the repo
-```
-cd $repo_name
-rm -rf .git   
-git init   
-git status
-```  
-You'll see a list of file, these are files that git doesn't recognize. At this point, feel free to change the directory names to match your project. i.e. change the parent directory Insight_Project_Framework and the project directory Insight_Project_Framework:
-Now commit these:
+#### Installation
 ```
-git add .
-git commit -m "Initial commit"
-git push origin $branch_name
+git clone https://github.com/hairong-wang/XLNet_learn2learn.git
+cd XLNet_learn2learn
 ```
+#### Requisites
+tensorflow-gpu==1.15.0-rc1
+absl-py==0.8.0
+Flask==1.1.1
+pip
 
-## Requisites
+#### Environment setup
+Everything needed fot the environment
 
-- List all packages and software needed to build the environment
-- This could include cloud command line tools (i.e. gsutil), package managers (i.e. conda), etc.
+## Steps to run
 
-#### Dependencies
+### Step1: Configuration 
 
-- [Streamlit](streamlit.io)
+### Step2: Prepare and Preprocess
+#### - Download dataset
+Download the dataset you want to use for finetuning
+The datasets used in this project are:
+- **The [Squad dataset](https://rajpurkar.github.io/SQuAD-explorer/) is used in this proeject.**
+- **The manual sampled and labeled AmazonQA and preprocessed dataset is available [here]**
 
-#### Installation
-To install the package above, pleae run:
-```shell
-pip install -r requiremnts
-```
-
-## Build Environment
-- Include instructions of how to launch scripts in the build subfolder
-- Build scripts can include shell scripts or python setup.py files
-- The purpose of these scripts is to build a standalone environment, for running the code in this repository
-- The environment can be for local use, or for use in a cloud environment
-- If using for a cloud environment, commands could include CLI tools from a cloud provider (i.e. gsutil from Google Cloud Platform)
-```
-# Example
+#### - Download model checkpoints
+- The model checkpoints is available [here]()
 
-# Step 1
-# Step 2
+#### - Convert dataset to SQuAD format(Optional)
+If you want to try other dataset, it needs to be converted to SQuAD format first.
 ```
-
-## Configs
-- We recommond using either .yaml or .txt for your config files, not .json
-- **DO NOT STORE CREDENTIALS IN THE CONFIG DIRECTORY!!**
-- If credentials are needed, use environment variables or HashiCorp's [Vault](https://www.vaultproject.io/)
-
-
-## Test
-- Include instructions for how to run all tests after the software is installed
+code for converting to squad
 ```
-# Example
-
-# Step 1
-# Step 2
+#### - Preprocess data
 ```
```

Comment:
It looks are you are still working on this README. I assume you will fill the necessary ones in the further PR.

Reply:
Yes, still working on, will try to finish asap, will submit a pr for that, thanks for your thorough review!